package.json

```diff
@@ -24,8 +24,8 @@
     "if-env": "^1.0.4",
     "import-sort-style-eslint": "^6.0.0",
     "lerna": "^2.11.0",
-    "node-addon-api": "1.7.2",
-    "node-addon-api-latest": "npm:node-addon-api@3.1.0",
+    "node-addon-api-1x": "npm:node-addon-api@1.7.2",
+    "node-addon-api": "3.1.0",
```

Comment:
as far as I understand 
you have added transitive dependencies to `devDependencies`
sorry, but I don't see how it can help
I believe the dependencies should be removed from `devDependencies` section